addons/point_of_sale/static/src/app/screens/ticket_screen/ticket_screen.js

```diff
@@ -425,7 +425,8 @@ export class TicketScreen extends Component {
             this.isDefaultOrderEmpty(order) ||
             order.payment_ids.some(
                 (payment) => payment.is_electronic() && payment.get_payment_status() === "done"
-            )
+            ) ||
+            this.getStatus(order) === _t("Paid")
```

Comment:
comparing with the return of `_t` seems a bit weird. Can you not just do `order.state != "draft"` ?